server/command.go

```diff
@@ -153,26 +153,26 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
 	var meetingID int
 	var createMeetingErr error
 
-	userPMISettingPref, getUserPMISettingErr := p.getPMISettingData(user.Id)
-	if getUserPMISettingErr != nil {
-		p.askUserForMeetingPreference(user.Id, args.ChannelId)
+	userPMISettingPref, err := p.getPMISettingData(user.Id)
+	if err != nil {
+		p.askPreferenceForThisMeeting(user.Id, args.ChannelId)
 		return "", nil
 	}
 
 	switch userPMISettingPref {
 	case zoomPMISettingValueAsk:
-		p.askUserForMeetingPreference(user.Id, args.ChannelId)
+		p.askPreferenceForThisMeeting(user.Id, args.ChannelId)
```

Comment:
Is this name given by them if not 
```suggestion
		p.askPreferenceForMeeting(user.Id, args.ChannelId)
```